lib/internal/crypto/keys.js

```diff
@@ -128,12 +130,33 @@ const [
   }
 
   const kAsymmetricKeyType = Symbol('kAsymmetricKeyType');
+  const kAsymmetricKeyDetails = Symbol('kAsymmetricKeyDetails');
+
+  // TODO: should the returne details object be frozen or otherwise protected
+  // from manipulation?
```

Comment:
thoughts?

Reply:
I wouldn't think so. Just adds a performance cost for little benefit.

Reply:
ok, removed the TODO comment